eessi_bot_job_manager.py

```diff
@@ -339,23 +330,16 @@ def process_finished_job(self, finished_job):
 
         # determine comment to be updated
         if "comment_id" not in finished_job:
-            comments = pull_request.get_issue_comments()
-            for comment in comments:
-                # NOTE adjust search string if format changed by event
-                #        handler (separate process running
-                #        eessi_bot_event_handler.py)
-                cms = f".*submitted.*job id `{finished_job['jobid']}`.*"
+            # search_pattern = f"submitted.*job id `{finished_job['jobid']}`"
+            finished_job_cmnt = get_comment(pull_request, finished_job['jobid'])
```

Comment:
If parameters to function `get_comment` are changed to `get_comment(pr, search_pattern)` the call here needs to be changed to.